3rdparty/CMakeLists.txt

```diff
@@ -18,9 +18,10 @@ add_subdirectory(glslang)
 set(SHADERC_SKIP_TESTS ON CACHE BOOL "Skip shaderc tests?" FORCE)
 set(SHADERC_SKIP_INSTALL ON CACHE BOOL "Install shaderc?" FORCE)
 # if it doesn't work without the `touch` on Linux, then fetch the latest submodule head of shaderc and try again
-#if (UNIX)
-#	file(WRITE ${THIRD_PARTY_SOURCE_DIR}/shaderc/libshaderc/libshaderc_combined.a "") #https://github.com/google/shaderc/issues/568
-#endif()
+# https://github.com/google/shaderc/issues/568
+if (UNIX)
+	file(WRITE ${THIRD_PARTY_SOURCE_DIR}/shaderc/libshaderc/libshaderc_combined.a "")
```

Comment:
I think this may fail if the output directory is not present there yet

Reply:
The folder is guaranteed by the submodule (this is a verified fix by Criss and shaderc's devs). Still a dirty work-around though but it's out of my support scope anyways.